contracts/examples/IncentivizedFlatKV.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./FlatDirectory.sol";
+
+contract IncentivizedFlatKV is FlatDirectory {
+    address public operator;
+
+    uint256 public immutable perChunkSize;
+    uint256 public immutable codeStakingPerChunk;
+
+    constructor(
+        uint8 slotLimit,
+        uint256 _perChunkSize,
+        uint256 _codeStakingPerChunk
+    ) payable FlatDirectory(slotLimit) {
+        perChunkSize = _perChunkSize;
```

Comment:
I guess we could make these two as constants because it is consensus parameters.